common-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '2.1'
 services:
   rebuild-swagger:
-    image: java
+    image: eclipse-temurin:latest
```

Comment:
I think this may default to latest without specifying. Doesn't hurt to be extra clear though